frontend/src/metabase/visualizations/components/ClickActions/ClickActionControl.tsx

```diff
@@ -31,6 +37,11 @@ export const ClickActionControl = ({
 
   const { buttonType } = action;
 
+  const handleClick =
+    isCustomClickAction(action) && action.onClick
+      ? () => (action as CustomClickAction).onClick?.(close)
```

Comment:
I'm just curious how you visualize the click action API would be. Now I only see it accepts `close`, but would we pass more props later on? If that's the case I think it's common to pass and object instead. e.g.
```suggestion
      ? () => (action as CustomClickAction).onClick?.({close})
```

So it's easier to extend later on.